man/org.freedesktop.systemd1.xml

```diff
@@ -4852,6 +4856,14 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       <varname>StateDirectory</varname>, <varname>CacheDirectory</varname> and <varname>LogsDirectory</varname>,
       which will create a symlink of the given name to the respective directory. The messages take an unused
       <varname>flags</varname> parameter, reserved for future backward-compatible changes.</para>
+
+      <para><varname>ExtraFileDescriptorNames</varname> contains file descriptor names passed to the service via
+      the ExtraFileDescriptors property in the StartTransient dbus API. The file descriptor names are passed to
+      the service manager via
+      <citerefentry><refentrytitle>sd_notify</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
```

Comment:
i don't grok this comment. sd_notify() is about passing stuff from service to service manager, bu i don't see how ExtraFileDescriptors would have anything to do with that?

instead it should probably reference $LISTEN_FDNAMES, which is how the service manager propagate sthe names to the service.

Reply:
Yea I meant to say to use the sd_listen_fd APIs. Instead of mentioning LISTEN_FDS, I referred the reader to sd_listen_fds doc.

```
         <para><varname>ExtraFileDescriptorNames</varname> contains file descriptor names passed to the service via
      the <varname>ExtraFileDescriptors</varname> property in the <function>StartTransientUnit()</function>
      method. See <citerefentry><refentrytitle>sd_listen_fds</refentrytitle><manvolnum>3</manvolnum></citerefentry>
      for more details on how to retrieve these file descriptors. While <varname>ExtraFileDescriptorNames</varname>
      contains the file descriptors and their names, <varname>ExtraFileDescriptorNames</varname> only contains
      names.</para>
```